src/solvers/file_solution_history.C

```diff
@@ -28,14 +28,34 @@
 namespace libMesh
 {
 
+/**
+   * Constructor, reference to system to be passed by user, set the
+   * stored_sols iterator to some initial value
+   */
+  FileSolutionHistory::FileSolutionHistory(System & system_)
+  : stored_sols(stored_solutions.end()),
+  _system(system_), localTimestamp(0),
+  timeTotimestamp()
+  {
+    dual_solution_copies.resize(system_.n_qois());
+
+    for(auto j : make_range(system_.n_qois()))
+    {
+      dual_solution_copies[j] = nullptr;
+    }
```

Comment:
Doesn't resize default-initialize the new entries to nullptr already?

Reply:
Fixed.